7.0-cli/Dockerfile

```diff
@@ -59,6 +59,8 @@ RUN ["chmod", "+x", "/docker-entrypoint.sh"]
 
 ENTRYPOINT ["/docker-entrypoint.sh"]
 
+RUN ["/docker-entrypoint.sh"]
```

Comment:
@tgerulaitis @punkstar the `test_runner.sh` ported from `docker-magento` fails when trying to install composer because the `php.ini` doesn't replace the memory placeholder. The placeholders are replaced in `docker-entrypoint.sh` can you see any problems with running it during the build?

Reply:
I don't think that's a good idea because running the entry point during the build will replace the placeholders which will stay replaced in the resulting image (afaik), making the placeholders moot.

The entry point should already be run on any command to the container (as indicated by the `ENTRYPOINT` instruction on line 60). It's best to figure out why its not replacing the placeholders - I suspect it's because there's no default value and you have to specify it with `-e PHP_MEMORY_LIMIT=x` when running each test.

Reply:
Good point. I'll revert to what I had previously.